app/forms/change_type_form.rb

```diff
@@ -1,29 +1,10 @@
 class ChangeTypeForm < Form
-  set_attributes_for :change, :job_termination, :new_job, :change_in_hours
+  set_attributes_for :change, :change_type
 
-  validate :at_least_one_change_type
+  validates_presence_of :change_type, message: "Choose one."
 
   def save
-    attributes_for(:change).each do |change_type, value|
-      if value == "1"
-        report.reported_changes.find_or_create_by(change_type: change_type)
-      elsif value == "0"
-        report.public_send("#{change_type}_change".to_sym)&.delete
-      end
-    end
-  end
-
-  def at_least_one_change_type
-    return true if job_termination == "1" || new_job == "1" || change_in_hours == "1"
-
-    errors.add(:change_type, "Please pick at least one change type.")
-  end
-
-  def self.existing_attributes(report)
-    hash = {}
-    report.reported_changes.each do |change|
-      hash[change.change_type] = "1"
-    end
-    HashWithIndifferentAccess.new(hash)
+    change_type = attributes_for(:change)[:change_type]
+    report.reported_changes.find_or_create_by(change_type: change_type)
```

Comment:
Shouldn't be anyway to choose an existing change, yet we are about to allow multiple changes of the same type soon, so just left this alone for now.